### PR TITLE
Add support for Triggers 2.0

### PR DESCRIPTION
**Pull request by kirre-bylund**

### CODE REVIEW

File: LootLockerSDK/Source/LootLockerSDK/Private/LootLockerGameEndpoints.cpp [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 #include "LootLockerGameEndpoints.h"
 
-FString ULootLockerGameEndpoints::GameBaseUrl = "https://{domainKey}api.lootlocker.io/game/";
+FString ULootLockerGameEndpoints::GameBaseUrl = "https://{domainKey}api.stage.internal.dev.lootlocker.cloud/game/";
```
Comment (JohannesLoot) [inline LootLockerSDK/Source/LootLockerSDK/Private/LootLockerGameEndpoints.cpp:5]: Nope!

**Review by JohannesLoot — CHANGES_REQUESTED** [review]
You accidentally commited the stage-url, change that before merging.
Other than that, all good!

File: LootLockerSDK/Source/LootLockerSDK/Private/LootLockerGameEndpoints.cpp (hunk 1 above)
Comment (kirre-bylund) [reply LootLockerSDK/Source/LootLockerSDK/Private/LootLockerGameEndpoints.cpp]: Fixed

**Review by JohannesLoot — APPROVED** [review]
All good! ✔️

**Pull request merged**